### ISSUE TITLE
Add 2.0 item sounds

### ISSUE THREAD

**Issue by FishyB** [FishyB/FactorioPlus#154]
[item] sounds from inventory, pickup, drop etc.

Just do a pass on everything, see what's missing something and then add the closest one that fits.
